test/graalvm/src/test/kotlin/GraalSmokeTest.kt

```diff
@@ -39,15 +39,6 @@ class GraalSmokeTest {
         future.get(1000, TimeUnit.MILLISECONDS)
     }
 
-    @Ignore("Raw mode is currently unsupported on native-image")
```

Comment:
If we remove the `assertThrows`, this test fails with `Inappropriate ioctl for device`

Actually, when using the native-build-tools plugins, every test is run two times. Once on the JVM to discover the tests, and a second time in native-image mode. It first fails when using JNA

```
com.sun.jna.LastErrorException: [25] Inappropriate ioctl for device
	at app//com.sun.jna.Native.invokeVoid(Native Method)
	at app//com.sun.jna.Function.invoke(Function.java:418)
	at app//com.sun.jna.Function.invoke(Function.java:364)
	at app//com.sun.jna.Library$Handler.invoke(Library.java:270)
	at app//com.github.ajalt.mordant.internal.syscalls.jna.$Proxy16.tcgetattr(Unknown Source)
	at app//com.github.ajalt.mordant.internal.syscalls.jna.SyscallHandlerJnaLinux.getStdinTermios(SyscallHandler.jna.linux.kt:86)
	at app//com.github.ajalt.mordant.internal.syscalls.SyscallHandlerPosix.enterRawMode(SyscallHandler.posix.kt:162)
	at app//com.github.ajalt.mordant.input.RawModeKt.enterRawMode(RawMode.kt:21)
	at app//com.github.ajalt.mordant.input.RawModeKt.enterRawMode$default(RawMode.kt:17)
	at app//com.github.ajalt.mordant.graalvm.GraalSmokeTest.raw mode test(GraalSmokeTest.kt:45)
	at java.base@21.0.3/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base@21.0.3/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base@21.0.3/java.util.ArrayList.forEach(ArrayList.java:1596)
```

But if we rewrite the test as 

```kotlin
    @Test
    fun `raw mode test`() {
        val imageCode = System.getProperty("org.graalvm.nativeimage.imagecode")
        val isNativeImage = imageCode == "buildtime" || imageCode == "runtime"
        if (!isNativeImage) {
            return
        }
        val t = Terminal(interactive = true)
        t.enterRawMode().use {}
    }
```

If fails the same way `Error reading terminal attributes`

I don't think it's possible to run this from a gradle test, so I removed it